release-1.6/release-1.6.md

```diff
@@ -0,0 +1,53 @@
+(To Be Approved at Dec 22 1.5 Retrospective Meeting)
+
+#Proposed timeline for 1.6
+Since Kubernetes 1.5 is set to release on Dec 12, 2016, and 
```

Comment:
@grodrigues3 yes, update please 1.5 string as this release has already happened.